gammapy/time/exptest.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+
+__all__ = [
+    'exptest_for_run',
+]
+
+def exptest_for_run(time_delta=[]):
+    """Compute Mr value, the level of variability for a certain period of time.
+
+    Longer description: A single Mr value can be calculated, which shows the level of variability
+                        for the whole period, or the Mr value for each run can be shown.
+
+
+    Ref:Prah(1999),http://adsabs.harvard.edu/abs/1999astro.ph..9399P
+
+    Parameters
+    ----------
+    run : list of int
```

Comment:
Update description of input parameters.
Now there's only one input called `time_delta`, not the parameters you list here.